### DISCUSSION TITLE
Empty Trip data

### DISCUSSION THREAD

**Discussion by sodgaard** [tillsteinbach/VWsFriend#215] [Q&A] [answered]
Hi Till,

I note that the trip data only contains time and duration. Is this normal?
![image](https://user-images.githubusercontent.com/44952678/162314917-86b53a50-8a99-49d7-94c7-1e5166b89df1.jpeg)

**Comment by tillsteinbach** [comment, marked as answer]
That depends on the car and booked services. For the MEB cars (ID3/4/5) you will not have the locations before the car gets the Firmware update 3.0 this summer. For older cars it should work if you purchased WeConnect Plus Package.

**Reply by sodgaard** [reply]
Thanks. Yes impatiently awaiting 2.4 and 3.0... :-)

What do you drive?

/Stephen

**Reply by tillsteinbach** [reply]
I drive a Passat GTE and my wife an ID.3

**Reply by sodgaard** [reply]
Nice. We now have an ID.3 pro S and a Tesla model Y...
